{{cookiecutter.project_slug}}/src/{{cookiecutter.project_slug}}/config.py

```diff
@@ -5,8 +5,8 @@
 try:
     from ._version import version as __version__
 except ImportError:
-    # broken installation, we don't even try unknown only works because we do poor mans
-    # version compare
+    # Broken installation, we don't even try.
```

Comment:
I don't get this...

Reply:
Let us remove it. I think I just copied it from pytest.

Reply:
What I realised only later: The `.version` - file is created automatically via `pyproject.toml`, right? Maybe we should document that somewhere? I am only guessing, for example :smile:

Reply:
The file is created by setuptools-scm when the project is installed. setuptools-scm computes the version number dynamically from the distance to the last tag in the repo. added a note.

Reply:
Great, thanks! Only thing is that we may want to be more specific what 

> when the project is installed

means precisely. I am pretty sure that `conda develop .` won't do the trick, how about `pip install -e`? Or only `python setup.py install` ? Is the latter relevant in the context of data analysis projects?

Reply:
The latter is the legacy approach (https://docs.python.org/3/install/) and pip is the recommended way. Apparently, conda develop does not perform an installation but extends your pythonpath. Let us never mention it again.

Reply:
Starting right there, then :smile: https://pytask-dev.readthedocs.io/en/latest/tutorials/how_to_set_up_a_project.html?highlight=setup.py#setup-py -- probably not worth waiting for the bigger overhaul.